Friends page and notification page

ceciliaccwei opened the pull request.

maharshmellow commented on src/components/notices/NoticesPage.jsx:
```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+import "../../styles/friends/FriendsPage.scss";
```
Why are we using the FriendsPage.scss file here?

ceciliaccwei replied:
The friends Page and the notification page are basically using the same style. I can rename the FriendsPage.scss to Friends-NoticesPage.scss

maharshmellow commented on src/styles/friends/FriendItem.scss:
```diff
@@ -0,0 +1,38 @@
+$accent-color: #4B84F2;
```
Use the accent colour from the common file

ceciliaccwei replied:
done! Also changed border color and background color in Notice/Friends Page to use common file😄

maharshmellow commented on src/components/friends/FriendsPage.jsx:
```diff
@@ -0,0 +1,71 @@
+                <p>Friends</p>
+                <p>
+                  (
+                  {friendsList.length}
```
Why are there parenthesis around this line on line 58 and 60

ceciliaccwei replied:
based on the figma design, the top left of the page will show [icon]Friends(30). The parenthesis are in the <p> tag.

maharshmellow replied:
Ok cool I didn’t see that 😀

ceciliaccwei commented:
merged notice and friends styles to one folder notices-friends since they share the Page.scss.
changed colours in Notices and Friends to use the ones in the common file.
fixed eslint.
